<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMainClass</class>
 <widget class="QMainWindow" name="FormMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLPI Data Extractor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Travail</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombre d'items</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarExecution">
        <property name="value">
         <number>24</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_AddJob">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset resource="formmain.qrc">
            <normaloff>:/FormMain/add22.png</normaloff>:/FormMain/add22.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_UpdateJob">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="icon">
           <iconset resource="formmain.qrc">
            <normaloff>:/FormMain/edit22.png</normaloff>:/FormMain/edit22.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_DeleteJob">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset resource="formmain.qrc">
            <normaloff>:/FormMain/Delete22.png</normaloff>:/FormMain/Delete22.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExecute">
          <property name="text">
           <string>Démarrer l'extraction</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>À &amp;propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="formmain.qrc"/>
  <include location="globalformressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>FormMainClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
